index.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
```

Comment:
Let's be a little more verbose... 
```suggestion
console.log("Fibonacci Sequence: " + getList(process.argv[2]));